ajax/libs/responsive/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "responsive",
+  "filename": "responsive.min.js",
+  "version": "1.0.5",
+  "description": "amCharts V3 Responsive plugin",
```

Comment:
What's your ref of `description`?
I think we can use https://github.com/amcharts/responsive/blob/master/responsive.js#L3 to update package.json.

Reply:
It's from homepage, but the one you mentioned seems better. I'll update it in a minute. Thanks for suggestion.